test/_linux.py

```diff
@@ -472,6 +480,73 @@ def test_procfs_path(self):
             psutil.PROCFS_PATH = "/proc"
             os.rmdir(tdir)
 
+    def test_psutil_is_reloadable(self):
+        imp.reload(psutil)
+
+    def test_no_procfs_for_import(self):
+        my_procfs = tempfile.mkdtemp()
+
+        with open(os.path.join(my_procfs, 'stat'), 'w') as f:
+            f.write('cpu   0 0 0 0 0 0 0 0 0 0\n')
+            f.write('cpu0  0 0 0 0 0 0 0 0 0 0\n')
+            f.write('cpu1  0 0 0 0 0 0 0 0 0 0\n')
+
+        self.assertNotAlmostEqual(psutil.cpu_percent(), 0)
+        self.assertNotAlmostEqual(sum(psutil.cpu_times_percent()), 0)
```

Comment:
There I was checking that my test further down would be valid – if the system as a whole reported no CPU usage (which seemed unlikely), then the `assertAlmostEqual` against `0` could spuriously pass even if it weren't using `my_procfs`. 

It's using `assertNotAlmostEqual` because the cpu times are reported as floats, and in the spurious pass case it might be that the system reported e.g. `0.00000001` CPU usage twice – which could be both `almostEqual` and `notEqual` to 0, providing a narrow path along which the test might pass even if the underlying functionality was broken.

It's a holdover from an older version of the test, though, since I now change `my_procfs` and ensure I can read that back, so I can remove it if you would like.